content/en/docs/tutorials/security/seccomp.md

```diff
@@ -275,17 +275,11 @@ Here's a manifest for that Pod:
 {{< codenew file="pods/security/seccomp/ga/audit-pod.yaml" >}}
 
 {{< note >}}
-The functional support for the already deprecated seccomp annotations
-`seccomp.security.alpha.kubernetes.io/pod` (for the whole pod) and
-`container.seccomp.security.alpha.kubernetes.io/[name]` (for a single container)
-is going to be removed with a future release of Kubernetes. Please always use
-the native API fields in favor of the annotations.
-
-Since Kubernetes v1.25, kubelets no longer support the annotations, use of the
-annotations in static pods is no longer supported, and the seccomp annotations
-are no longer auto-populated when pods with seccomp fields are created.
-Auto-population of the seccomp fields from the annotations is planned to be
-removed in a future release.
+Older versions of Kubernetes allowed you to configure seccomp
```

Comment:
Should we mention `Older versions before v1.26`?

Reply:
Because the deprecation was so long ago, I recommend omitting it: we don't support any version of Kubernetes where these annotations aren't already deprecated, and we haven't done for a while.

I know that end users do use older Kubernetes. However, we should try to provide docs that are relevant for the version we're really supporting.

Related: it'd be nice to have a widget that highlights the different available docs version. If we had that, this would be a good place to include it.

Reply:
> Because the deprecation was so long ago, I recommend omitting it: we don't support any version of Kubernetes where these annotations aren't already deprecated, and we haven't done for a while.
> 
> I know that end users do use older Kubernetes. However, we should try to provide docs that are relevant for the version we're really supporting.
> 
> Related: it'd be nice to have a widget that highlights the different available docs version. If we had that, this would be a good place to include it.

It's SGTM